### ISSUE TITLE
wxWidgets fails

### ISSUE THREAD

**Issue by remontees** [firestrand/phatch#2]
Hello,

There are some bugs using Linux Mint 19.1

  /usr/lib/python2.7/dist-packages/wx-3.0-gtk3/wx/_core.py:8196: Warning: cannot register existing type 'GdkDisplayManager'
  return _core_.PyApp__BootstrapApp(*args, **kwargs)
/usr/lib/python2.7/dist-packages/wx-3.0-gtk3/wx/_core.py:8196: Warning: g_once_init_leave: assertion 'result != 0' failed
  return _core_.PyApp__BootstrapApp(*args, **kwargs)
/usr/lib/python2.7/dist-packages/wx-3.0-gtk3/wx/_core.py:8196: Warning: g_object_new_with_properties: assertion 'G_TYPE_IS_OBJECT (object_type)' failed
  return _core_.PyApp__BootstrapApp(*args, **kwargs)

**Comment by elvisef** [comment]
Yep, I have these bugs as well while starting Phatch, so finall it cannot be started...
I am using Debian 10 (Buster).

![2019-12-12_05-48](https://user-images.githubusercontent.com/25976749/70683574-2721e080-1ca3-11ea-9796-7d0b6efdfee9.png)

**Comment by firestrand** [comment]
Thanks for reporting this. Phatch has undergone substantial updates since this 2019 report, including migration from Python 2.7/wxPython 3 to Python 3/wxPython 4 and fixes for Linux GUI behavior. The current prerelease is v0.4.0-rc.2: https://github.com/firestrand/phatch/releases/tag/v0.4.0-rc.2

We have verified the updated application and full test gate on Ubuntu 24.04 ARM64. We have not reproduced this exact historical failure on Mint 19.1 or Debian 10, so this closure is not a claim that its specific root cause was confirmed fixed. We are closing the old report because its dependency stack has been superseded.

Please reopen if the latest version causes startup or other problems. If GitHub does not allow you to reopen, comment here or open a new issue linking this one. Include your distribution/version, CPU architecture, Phatch version, Python and wxPython versions, exact installation and launch commands, and complete error output. The release application ZIP is macOS-only; Linux testing should use the updated source, with the required native wxPython build dependencies where applicable.
